src/main/java/uk/gov/hmcts/ccd/domain/service/aggregated/MergeDataToSearchResultOperation.java

```diff
@@ -72,34 +72,27 @@ public SearchResultView execute(final CaseType caseType,
 
     private SearchResultViewColumn createSearchResultViewColumn(final SearchResultField searchResultField, final CaseField caseField) {
         return new SearchResultViewColumn(
-            buildCaseFieldId(searchResultField),
+            searchResultField.buildCaseFieldId(),
             buildCaseFieldType(searchResultField, caseField),
             searchResultField.getLabel(),
             searchResultField.getDisplayOrder(),
             searchResultField.isMetadata());
     }
 
     private boolean filterDistinctFieldsByRole(final HashSet<String> addedFields, final SearchResultField resultField) {
-        if (addedFields.contains(resultField.getCaseFieldId())) {
+        String id = resultField.buildCaseFieldId();
```

Comment:
can we have SearchResultField and CriteriaInput use the same method name please?